src/store/thunk.js

```diff
@@ -0,0 +1,48 @@
+import {
+  LOAD_MOVIES,
+  LOAD_MOVIES_BY_DATE,
+  LOAD_MOVIES_BY_RATING,
+  LOAD_MOVIES_BY_TITLE,
+} from "./actionTypes";
+
+export const loadMovies = () => async (dispatch) => {
+  const apiLink = "http://localhost:4000/movies?limit=100";
```

Comment:
move 'http://localhost:4000/movies' to a constant

Reply:
Fixed